src/app/menu/menu.component.scss

```diff
@@ -0,0 +1,212 @@
+$blue: #04529D;
+$outer-circle-left-pos: 91%;
+$outer-circle-top-pos: -2%;
+
+$hex-circle-left-pos: -8%;
+$hex-circle-top-pos: -4%;
+
+$intro-color: #00F977;
+$malaria-color: #EB5A61;
+$med-color: #017E84;
+$pc-policy-color: #F2AE3E;
+$unlocked-color: #ffffff;
```

Comment:
how is this different from `$intro-outer-circle-color` & `$unlocked-outer-circle-color`?

you are defining variables so that they can be reused, not duplicated